backend/routes/postReaction.py

```
@@ -25,207 +25,180 @@ def get_db():
     finally:
         db.close()
 
+
+# Helper: notify user if not the actor
+
+def notify_if_not_self(db, actor_id, recipient_id, notif_type, post_id):
```

Ruff (0.8.2)

Missing return type annotation for public function `notify_if_not_self`

Add return type annotation: `None`

(ANN201)